src/dbnode/storage/bootstrap.go

```diff
@@ -185,6 +204,10 @@ func (m *bootstrapManager) Bootstrap() (BootstrapResult, error) {
 	m.lastBootstrapCompletionTime = xtime.ToUnixNano(m.nowFn())
 	m.state = Bootstrapped
 	m.Unlock()
+	asyncResult.bootstrapResultFn = func() BootstrapResult {
```

Comment:
nit: Are we planning on executing any logic here? Could just set the result value?

Reply:
Probably not for now, will update to set the result value directly.